src/raglite/_database.py

```diff
@@ -91,8 +91,9 @@ def from_body(
         document_id: DocumentId, index: int, body: str, headings: str = "", **kwargs: Any
     ) -> "Chunk":
         """Create a chunk from Markdown."""
+        combined_data = body.encode() + document_id.encode() + str(index).encode()
         return Chunk(
-            id=hash_bytes(body.encode()),
+            id=hash_bytes(combined_data),
```

Comment:
I think we can drop the `body.encode()` component as it should already be a unique id given the `document_id` and `index`. And then since it's so simple I would run the computation inline as `id=hash_bytes(document_id.encode() + str(index).encode())`.

Alternatively, we could do `id=hash_bytes(f"{document_id}-{index}".encode())`, which is even more compact.